src/index.html

```diff
@@ -9,10 +9,30 @@
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <title>MA-task-layout_poduct-cards</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
 </head>
 <body>
+  <div data-qa="card" class="card">
+    <img class="product-photo" src="./images/imac.jpeg" alt="photo Moноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)">
```

Comment:
Attr alt should be concise
```suggestion
    <img
      class="product-photo"
      src="./images/imac.jpeg"
      alt="Apple IMac"
    >
```